CHANGELOG.md

```diff
@@ -0,0 +1,46 @@
+# Changelog
+
+## [Unreleased]
+
+## v0.2.0 - 2025-04-21
+
+### Breaking Changes
+
+- We now use [traitlets](https://traitlets.readthedocs.io/en/stable/) to handle configuration and command line arguments. Part of [PR #22](https://github.com/2i2c-org/jupyterhub-home-nfs/pull/22). This is a breaking change in how Helm values are passed in:
+
+  For example, the path, hard quota limits etc are now passed in `quotaEnforcer.config.QuotaManager` instead of directly under `quotaEnforcer`. For example:
+
+  ```yaml
+  quotaEnforcer:
+  config:
+    QuotaManager:
+      paths: ["/export"]
+      hard_quota: 10
+  ```
+
+- As part of [PR #22](https://github.com/2i2c-org/jupyterhub-home-nfs/pull/22), the default persistent volume size label was updated from 10Gi to 1M to clarify that it's a placeholder and not the actual disk size.
+  For existing deployments, if the size was previously unset, it must now be explicitly set to the current value (e.g. `10Gi`) like this:
+
+      ```yaml
```

Comment:
Where would this config be nested under?

Reply:
This is a top level config. I tweaked the text a bit to make that clearer.